packages/core/src/config/config.ts

```diff
@@ -46,6 +46,16 @@ function getDomains() {
   return domains;
 }
 
+function getIgnoreFilePath(configFile?: string): string | undefined {
+  if (configFile) {
+    return doesConfigFileExist(configFile)
```

Comment:
Can we also check if the ignore file exists here?

Reply:
I think that is doesn't make any sense here - because we checking or compiling path for the `ignoreFile`, check for file is added in `resolveIgnore` method.